apiserver/plane/app/views/cycle/base.py

```diff
@@ -71,6 +74,72 @@ def get_queryset(self):
             project_id=self.kwargs.get("project_id"),
             workspace__slug=self.kwargs.get("slug"),
         )
+        cancelled_issues = (
+            CycleIssue.objects.filter(
```

Comment:
Can we use `Issue` here instead of CycleIssue as Issue already has a manager ?

Reply:
Yes. Changed to issue manager.